contribs/gmf/src/directives/filterselector.js

```diff
@@ -266,6 +274,14 @@ gmf.FilterselectorController = class {
     } else {
       this.filtrableLayerNodeNames_ = null;
     }
+    if (usrFunc &&
+        usrFunc['preset_layer_filter'] &&
+        usrFunc['preset_layer_filter'][0]
+    ) {
+      this.defaultFiltrableDataSourceName_ = usrFunc['preset_layer_filter'][0];
+    } else {
+      this.defaultFiltrableDataSourceName_ = undefined;
+    }
```

Comment:
You can try it, that would set an `undefined` value. But fine to me to keep as you did.